index.html

```diff
@@ -122,177 +68,185 @@
-        </li>
-        <li>
-          <div>
-            <i class="fal fa-keynote fa-2x"></i>
-            <h3>WHO Ignite</h3>
+      <div class="fordesktopnav">
+        <ul class="accounttopbar">
+          <li>
+            <a href="#">
+              <i class="fab fa-facebook-f"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li>
+            <a class="child3" href="#"> English </a>
+          </li>
+          <li>
+            <a href="#"> My Page </a>
+          </li>
+          <li>
+            <a href="#"> Logout </a>
+          </li>
+        </ul>
+        <nav class="nav-bar">
+          <a class="logo" href="#">
+            <img src="images/h-logo-white.svg" alt="logo" />
+          </a>
+          <div class="mainmenunav">
+            <ul class="navmenu">
+              <li>
+                <a class="current-page" href="#"> Home </a>
+              </li>
+
+              <li>
+                <a href="pages/about.html"> About </a>
+              </li>
+              <li>
+                <a href="#"> Join </a>
+              </li>
+              <li>
+                <a href="#"> Sponsor </a>
+              </li>
+              <li>
+                <a href="#"> News </a>
+              </li>
+            </ul>
+            <button type="button" class="compaign-button button">
+              WHO Campaign
+            </button>
           </div>
-
-          <p>The Proceedings is to enable dialog and not constitute an verbatim record of the proceedings.</p>
-        </li>
-      </ul>
-      <a class="see-more">SEE THE WHOLE PROGRAM</a>
-    </section>
-    <section class="speakers">
-      <div class="title-group">
-        <h2 class="title">Featured speakers</h2>
-        <hr>
-      </div>
-      <ul class="dynamic-speakers">
-      </ul>
-    </section>
-    <section class="partners">
-      <div class="title-group">
-        <h2 class="title">Partner</h2>
-        <hr>
+        </nav>
       </div>
-      <ul>
-        <li>
-          
-          HEPA Europe
-        </li>
-        <li>
-          
-         NITAGs
-        </li>
-        <li>
-          
-         NITAGs
-        </li>
-        <li>
-         SEEHN
-        </li>
-        <li>
-         ETAGE
-        </li>
-      </ul>
-    </section>
-  </main>
-
-  <footer class="license">
-
-    <a class="logo" href="#">
-      <img src="images/h-logo-white.svg" alt="logo">
-       
-    </a>
-    <p class="logo-text">
-      2022 World health Assembly. All Rights Reserved.
-      This is registred under the MIT license. for more
-      info, please <a
-        href="https://github.com/Alusp/World-Health-Assembly">See
-        here</a>
-    </p>
-
-  </footer>
-
-  <script src="script.js"></script>
-
-</body>
-
+    </header>
+
+    <main>
+      <section class="head">
+        <div class="head-bg"></div>
```

Comment:
- [ ] The background you chose for the Home and About page `head` section are interesting but it makes it hard to read it's content. Try applying a lighter filter to improve the contrast with the dark text colors.